src/components/Auth/Login/Login.react.js

```diff
@@ -156,7 +153,7 @@ class Login extends Component {
 		else {
 			this.passwordErrorMessage = '';
 		}
-		if (!state.emailError && !state.passwordError && this.state.password != '') {
+		if (!state.emailError && !state.passwordError) {
```

Comment:
@anshumanv The bug with login page occurred due to this line. It was implemented to keep the button disabled if password field is empty and since auto fill does not change the state therefore the bug occurred. I am removing it for now.

Reply:
![image](https://user-images.githubusercontent.com/21009455/40994901-37c60be2-691b-11e8-8916-66e17f5eea4e.png)
I'm still facing this but it's a minor issue which can be fixed in subsequent PRs, great work! :+1: